INTITA/Modules/Profile/Cells/ProfileTableViewCell.swift

```diff
@@ -7,23 +7,34 @@
 
 import UIKit
 
+protocol ProfileTableViewCellDelegate: AnyObject {
+    func goToVC(number: Int)
+}
+
 class ProfileTableViewCell: UITableViewCell {
 
     @IBOutlet weak var container: UIView!
     @IBOutlet weak var button: UIButton!
     @IBOutlet weak var label: UILabel!
     
+    var row: Int?
```

Comment:
Is there a necessity to make it optional? Row could be absent?

Reply:
If make it non-optional it is necessary to provide a constructor. I want to use tableView.dequeReusableCell() that's why row is optional.